### DISCUSSION TITLE
Deploying an app over two domains

### DISCUSSION THREAD

**Discussion by abbbdab** [remix-run/remix#4997] [Q&A]
Hi all, 

I am trying to build a product, and the idea is I'll have a marketing page on say www.XYZ.com, and the product / dashboard on app.XYZ.com.

How can I achieve that? I want to keep same repository, and split it between marketing and app folders and use pathless layout etc. to handle the routing. I am thinking of deploying it on Vercel. 

Any help will be appreciated. 

Thanks.

**Comment by sergiodxa** [comment]
Create a folder for one app, and another for the other app

```
/web
  /app <- Remix app folder
  /package.json
/dashboard
  /app <- Remix app folder
  /package.json
```

Then deploy each folder as individual apps.

**Reply by abbbdab** [reply]
Thanks! I’ll give it a go
